examples/blog/astro.config.mjs

```diff
@@ -4,7 +4,4 @@ import preact from '@astrojs/preact';
 // https://astro.build/config
 export default defineConfig({
 	integrations: [preact()],
-	buildOptions: {
-		site: 'https://example.com/',
-	},
```

Comment:
Nit: not sure why we're removing this? Seems out-of-scope for this PR 🤷

Reply:
It is! Thanks for catching, I had been having trouble this during testing and forgot to add it back after the issue was fixed